core/domain/email_manager_test.py

```diff
@@ -1742,6 +1821,22 @@ def test_that_feedback_message_emails_are_correct(self) -> None:
                 sent_email_model.intent,
                 feconf.EMAIL_INTENT_FEEDBACK_MESSAGE_NOTIFICATION)
 
+    def test_feedback_message_emails_are_not_sent_if_user_cannot_recieve_email(
```

Comment:
```suggestion
    def test_feedback_message_emails_are_not_sent_if_user_cannot_receive_email(
```

recieve -> receive

ditto elsewhere

Reply:
Fixed